[
  {
    "Id": "834309",
    "ThreadId": "355178",
    "Html": "\r\n<p>Hi,<br>\r\n<br>\r\nI modified the source to include a call to AllowPartiallyTrustedCallers and strongly sign the assembly. My application has to run in a sandbox so my permissions are a bit tight. However, I am getting a weird exception that doesn't makes sense to me. Anyone\r\n else encounter this?</p>\r\n<p>When creating a new GraphLayout&lt;A,B,C&gt; I get&nbsp;<br>\r\n</p>\r\n<pre>System.Windows.Markup.XamlParseException occurred\r\n  HResult=-2146233087\r\n  Message='The invocation of the constructor on type 'UserControl.MyGraphLayout' that matches the specified binding constraints threw an exception.' Line number '197' and line position '14'.\r\n  Source=PresentationFramework\r\n  LineNumber=197\r\n  LinePosition=14\r\n  StackTrace:\r\n       at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)\r\n       at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)\r\n       at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)\r\n       at System.Windows.Application.LoadComponent(Object component, Uri resourceLocator)\r\n       at UserControl.GraphUserControl.InitializeComponent()\r\n       at UserControl.GraphUserControl..ctor()\r\n  InnerException: System.TypeInitializationException\r\n       HResult=-2146233036\r\n       Message=The type initializer for 'GraphSharp.Controls.GraphLayout`3' threw an exception.\r\n       Source=GraphSharp.Controls\r\n       TypeName=GraphSharp.Controls.GraphLayout`3\r\n       StackTrace:\r\n            at GraphSharp.Controls.GraphLayout`3..ctor()\r\n            at Stru2.UserControls.ScenarioUserControl.MyGraphLayout..ctor()\r\n       InnerException: System.Security.VerificationException\r\n            HResult=-2146233075\r\n            Message=Operation could destabilize the runtime.\r\n            Source=GraphSharp.Controls\r\n            StackTrace:\r\n                 at GraphSharp.Controls.GraphLayout`3..cctor()\r\n            InnerException: \r\n</pre>\r\n<p></p>\r\n",
    "PostedDate": "2012-05-09T15:41:25.857-07:00",
    "UserRole": null,
    "MarkedAsAnswerDate": null
  }
]